components/ActivityPage.tsx

```diff
@@ -1,10 +1,236 @@
-import React from 'react';
-import { View, Text } from 'react-native';
+import React, { useState, ChangeEvent } from 'react';
+import RNDateTimePicker from '@react-native-community/datetimepicker';
+import { daysOfTheWeek } from '../utils/constants';
+import {
+  StyleSheet,
+  View,
+  ViewStyle,
+  Text,
+  TextStyle,
+  TextInput,
+  TouchableOpacity
+} from 'react-native';
+
+interface Styles {
+  title: TextStyle;
+  container: ViewStyle;
+  label: TextStyle;
+  input: TextStyle;
+  inputWrapper: ViewStyle;
+  button: ViewStyle;
+  daysOfTheWeekContainer: ViewStyle;
+  selectedDaysOfWeek: ViewStyle;
+}
+
+interface IValues {
+  name?: string;
+  description?: string;
+  motivation?: string;
+  reminder?: string;
+  selectedDays?: any;
+  date?: any;
+}
+interface IProps {
```

Comment:
Love the types 👏🏽 

However rename them to something appropriate. For example, this could be ActivityPageProps